website/content/docs/auth/login-mfa/index.mdx

```diff
@@ -43,7 +43,7 @@ MFA in Vault includes the following login types:
 TOTP passcodes by default. We recommend that per-client [rate limits](/docs/concepts/resource-quotas)
 are applied to the relevant login and/or mfa paths (e.g. `/sys/mfa/validate`). External MFA
 methods (`Duo`, `Ping` and `Okta`) may already provide configurable rate limiting. Rate limiting of
-Login MFA paths will be enforced by default in a future release.
+Login MFA paths are enforced by default in Vault 1.10.1 and above.
```

Comment:
Isn't it in 1.10.0 and above?

Reply:
Ah, I didn't see that this line is part of an existing sentence, due to the line break in the Markdown. I thought it was starting a new sentence.